http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -894,13 +899,9 @@ class LeoPubsubMessageSubscriber[F[_]: Timer: ContextShift: Parallel](
       task = for {
```

Comment:
Out of scope for this PR, but I recall thinking this method could be cleaned up

Reply:
Should I make a note in the code like a TODO or a ticket?

Reply:
Eh, not sure what concretely to suggest so don't worry about it :)